src/test/app/Vault_test.cpp

```diff
@@ -1888,6 +1888,37 @@ class Vault_test : public beast::unit_test::suite
                     env(tx, ter(tecNO_AUTH));
```

Comment:
It's hard to tell with your test case/suite structure but do you have tests where the issuer is the depositor? Also you test frozen but I don't see deep freeze or auth or no ripple?

Reply:
In this branch, there is withdrawal to 3rd party which happens to be issuer  https://github.com/XRPLF/rippled/blob/fd178c550b434534209d84e700ffa0c7ebdfca74/src/test/app/Vault_test.cpp#L376

However this branch does not allow the issuer to be also depositor. This is fixed in #5518 and the test to match is https://github.com/XRPLF/rippled/blob/42be446f6979299a2c4eee58514c823b578af4ac/src/test/app/Vault_test.cpp#L399